tests/opentelemetry-test-utils/src/opentelemetry/test/test_base.py

```diff
@@ -85,6 +90,79 @@ def sorted_spans(self, spans):  # pylint: disable=R0201
             reverse=True,
         )
 
+    def sorted_metrics(self, metrics):  # pylint: disable=R0201
+        """
+        Sorts metrics by metric name.
+        """
+        return sorted(
+            metrics,
+            key=lambda m: m.name,
+        )
+
+    def get_sorted_metrics(self):
+        resource_metrics = (
+            self.memory_metrics_reader.get_metrics_data().resource_metrics
+        )
+
+        all_metrics = []
+        for metrics in resource_metrics:
+            for scope_metrics in metrics.scope_metrics:
+                all_metrics.extend(scope_metrics.metrics)
+
+        return self.sorted_metrics(all_metrics)
+
+    def assert_histogram_expected(self, data_point, expected_value):
+        self.assertEqual(
+            data_point.count,
+            1,
+        )
+        self.assertEqual(
+            data_point.sum,
+            expected_value,
+        )
+        self.assertEqual(
+            data_point.max,
+            expected_value,
+        )
+        self.assertEqual(
+            data_point.min,
+            expected_value,
+        )
```

Comment:
This doesn't look right to me. Why would you expect the `expected_value` to match all three `min`, `max`, and `sum`? I would have assumed it takes an expected `HistogramDataPoint` and makes the assertions.